apps/i18n/signup/en_us.json

```diff
@@ -14,9 +14,11 @@
   "sign_up_as_a_student": "Sign up as a student",
   "sign_up_as_a_teacher": "Sign up as a teacher",
   "free_curriculum_forever": "Free curriculum. Forever.",
-  "read_our_commitment": "[Read our commitment]({commitmentLink}) to keeping our curriculum offerings free for everyone.",
+  "read_our_commitment": "Read our commitment to keeping our curriculum offerings free for everyone.",
```

Comment:
Is this one of the cases where we'd need a new string because the interpolation changed?

Reply:
Oh you're right, great catch!